packages/components/src/components/shell-panel/shell-panel.tsx

```diff
@@ -115,7 +115,7 @@ export class ShellPanel extends LitElement {
   /**
    * Specifies the component's position. Will be flipped when the element direction is right-to-left (`"rtl"`).
    *
-   * @deprecated No longer necessary.
+   * @deprecated  in v4.0, removal target v6.0 -  No longer necessary.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated  in v4.0.0, removal target v6.0.0 -  No longer necessary.
```